_pages/clients.md

```diff
@@ -31,14 +31,13 @@ redirect_from:
   </div>
 </div>
 
-After you learn how to call algorithms, check out our <a href="{{site.baseurl}}/data">Data Connectors</a> that we support and learn how to work with data using our <a href="http://docs.algorithmia.com/">Data API</a>.
+After you've made your first API call, you can learn more about using the Algorithmia clients in the guides below. The clients can be used in your own applications to call algorithms, [access data]({{site.baseurl}}/data), or to manage the Algorithmia platform. The clients can also be used in the algorithms you write, letting you work with the [Data API](https://algorithmia.com/developers/api/#data) or call additional algorithms from your own code, letting you use utility functions and chain algorithms together. 
```

Comment:
lets add the link to getting started https://algorithmia.com/developers/getting-started for "made your first API call"